docs/compatibility.md

```diff
@@ -2,6 +2,12 @@
 
 This document provides detailed descriptions of the BC-breaking changes in MMDetection3D.
 
+## MMDetection3D 0.15.0
+
+### Enhance `IndoorPatchPointSample` transform
+
+We enhance the pipeline function `IndoorPatchPointSample` by adding more choices for patch selection and removing a useless parameter `sample_rate`. Please modify the code as well as the config files accordingly if you use this transform.
```

Comment:
We plan to remove the unused parameter `sample_rate` in the future.

Reply:
Modified.